src/Model/Contact.php

```diff
@@ -724,6 +725,7 @@ public static function updateSelfFromUserID($uid, $update_avatar = false)
 				'thumb' => DI::baseUrl() . '/photo/avatar/' . $uid .'.' . $file_suffix];
 			DBA::update('profile', $fields, ['uid' => $uid]);
 		}
+		return $update;
```

Comment:
Code standards: Please add a new line after closing condition braces.
```suggestion
		}

		return $update;
```